src/Microsoft.Azure.WebJobs.Extensions.PostgreSql/PostgreSqlBindingUtilities.cs

```diff
@@ -0,0 +1,314 @@
+        /// as in "@param1=,@param2=param2"
+        /// </summary>
+        /// <param name="parameters">The parameter string to be parsed</param>
+        /// <param name="command">The SqlCommand to which the parsed parameters will be added to</param>
+        /// <exception cref="ArgumentNullException">
+        /// Thrown if command is null
+        /// </exception>
+        public static void ParseParameters(string parameters, NpgsqlCommand command)
+        {
+            Console.WriteLine("ParseParameters");
+            Console.WriteLine(parameters);
+            if (command == null)
+            {
+                throw new ArgumentNullException(nameof(command));
+            }
+
+            // If parameters is null, user did not specify any parameters in their function so nothing to parse
+            if (!string.IsNullOrEmpty(parameters))
+            {
+                // Because we remove empty entries, we will ignore any commas that appear at the beginning/end of the parameter list,
+                // as well as extra commas that appear between parameter pairs.
+                // I.e., ",,@param1=param1,,@param2=param2,,," will be parsed just like "@param1=param1,@param2=param2" is.
+                string[] paramPairs = parameters.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries);
+
+                foreach (string pair in paramPairs)
+                {
+                    // Note that we don't throw away empty entries here, so a parameter pair that looks like "=@param1=param1"
+                    // or "@param2=param2=" is considered malformed
+                    string[] items = pair.Split('=');
+                    if (items.Length != 2)
+                    {
+                        throw new ArgumentException("Parameters must be separated by \",\" and parameter name and parameter value must be separated by \"=\", " +
+                           "i.e. \"@param1=param1,@param2=param2\". To specify a null value, use null, as in \"@param1=null,@param2=param2\"." +
+                           "To specify an empty string as a value, simply do not add anything after the equals sign, as in \"@param1=,@param2=param2\".");
+                    }
+                    if (!items[0].StartsWith("@", StringComparison.InvariantCultureIgnoreCase))
+                    {
+                        throw new ArgumentException("Parameter name must start with \"@\", i.e. \"@param1=param1,@param2=param2\"");
+                    }
+
+
+                    if (items[1].Equals("null", StringComparison.OrdinalIgnoreCase))
+                    {
+                        NpgsqlParameter parameter = new NpgsqlParameter(items[0], NpgsqlDbType.Text); // change NpgsqlDbType to appropriate type
+                        parameter.Value = DBNull.Value;
+                        command.Parameters.Add(parameter);
+                    }
```

Comment:
I wanted to make sure if we are handling empty string and null value correctly here since they are very different. I didn't dig deeper on this but wanted to give heads up on that.

Reply:
Changed this function to make it clearer when string.Empty, DBNull.Value, or a value will be inserted.
string.Empty will only be assigned when the parameter is missing an equals sign like "@param1" or when there is no value after the equals like "@param1="
DBNull.Value is only assigned when the string 'null' is explicitly passed as the value, for example, '@param1=null'.